### ISSUE TITLE
Django 2.x - No module named 'django.core.urlresolvers'

### ISSUE THREAD

**Issue by br0th3r** [jrief/django-angular#327]
File "/home/www.django/incoan/env/lib/python3.5/site-packages/djng/forms/fields.py", line 13, in <module>
    from django.core.urlresolvers import reverse_lazy
ImportError: No module named 'django.core.urlresolvers'

SOLUTION: Change "django.core.urlresolvers" to "django.urls"

**Comment by diogosimao** [comment]
It's the only change that must be done in order to get things working with Django 2.0, isn't it?

**Comment by JLashmar** [comment]
No, you also have to change core/urlresolvers.py as well, after that it worked okay for me

**Comment by Cyber1408** [comment]
use from django.urls import reverse

**Comment by pramod870** [comment]
I also used from django.urls import reverse but still got error

**Comment by abhinav7879** [comment]
i also used django.urls but still getting error
